corehq/apps/users/tasks.py

```diff
@@ -41,6 +42,13 @@ def bulk_upload_async(domain, user_specs, group_specs, location_specs):
         'messages': results
     }
 
+
+def run_case_post_save_signals(docs):
+    for doc in docs:
+        case = CommCareCase.wrap(doc)
+        case_post_save.send(CommCareCase, case=case)
```

Comment:
i think it would be better to make a new signal for this and connect the reminders to both. I don't know if `create_case_repeat_records` is smart enough to check for deleted cases, and even if it is, it seems unlikely that anyone in the future grabbing the post save signal would expect to be passed deleted casees sometimes